dss/operations/events.py

```diff
@@ -86,24 +88,49 @@ def forward_keys(bundle_fqids):
 @events.action("journal",
                arguments={"--prefix": dict(required=True,
                                            help="flashflood prefix to journal events"),
-                          "--minimum-number-of-events": dict(default=1000, type=int),
-                          "--minimum-size": dict(default=1024 * 1024, type=int)})
+                          "--number-of-events": dict(default=None, type=int),
+                          "--starting-journal-id": dict(default=None),
+                          "--job-id": dict(default=None)})
 def journal(argv: typing.List[str], args: argparse.Namespace):
```

Comment:
What is the high-level idea behind this `dss-ops.py events journal` action?

Reply:
When a new event occurs, the event data is stored as an s3 object. During journaling, these objects are compiled into a _journal_ of many events and stored as a single s3 object. This makes event replay faster.